runtime/android/core/src/org/xwalk/core/XWalkCoreWrapper.java

```diff
@@ -145,11 +152,30 @@ private boolean findSharedCore() {
         XWalkApplication application = XWalkApplication.getApplication();
         if (application == null) Assert.fail("Must use or extend XWalkApplication");
 
+        if (!XWalkSdkVersion.VERIFY_XWALK_APK) {
+            Log.d(TAG, "Do not verify the xwalk package");
```

Comment:
The message here can be improved. How about "Not verifying Crosswalk runtime package integrity."?